src/index.js

```diff
@@ -3,6 +3,9 @@ import ReactDOM from 'react-dom'
 
 import App from './containers/App'
 import * as serviceWorker from './serviceWorker'
+import AuthProvider from './providers/authProvider'
+
+AuthProvider.initOauth()
```

Comment:
This function is called on this file, which means on every screen we call  the`iniOAuth` method and this method is called over and over and over